Formula/b/bloaty.rb

```diff
@@ -25,21 +36,16 @@ class Bloaty < Formula
   depends_on "protobuf"
   depends_on "re2"
 
-  # Support system Abseil. Needed for Protobuf 22+.
-  # Backport of: https://github.com/google/bloaty/pull/347
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/86c6fb2837e5b96e073e1ee5a51172131d2612d9/bloaty/system-abseil.patch"
-    sha256 "d200e08c96985539795e13d69673ba48deadfb61a262bdf49a226863c65525a7"
-  end
-
   def install
     # https://github.com/protocolbuffers/protobuf/issues/9947
     ENV.append_to_cflags "-DNDEBUG"
     # Remove vendored dependencies
     %w[abseil-cpp capstone protobuf re2].each { |dir| rm_r(buildpath/"third_party"/dir) }
     abseil_cxx_standard = 17 # Keep in sync with C++ standard in abseil.rb
-    inreplace "CMakeLists.txt", "CMAKE_CXX_STANDARD 11", "CMAKE_CXX_STANDARD #{abseil_cxx_standard}"
-    inreplace "CMakeLists.txt", "-std=c++11", "-std=c++17"
+    stable do
```

Comment:
I am bit surprised that `stable do` also works, but this might be preferred
```suggestion
    if build.stable?
```

Reply:
I'm happy to make this change. However, I was extrapolating from https://docs.brew.sh/Formula-Cookbook#patches where it says:

> [patch](https://rubydoc.brew.sh/Formula#patch-class_method)es can be declared in [stable](https://rubydoc.brew.sh/Formula#stable-class_method) and [head](https://rubydoc.brew.sh/Formula#head-class_method) blocks. Always use a block instead of a conditional, i.e. stable do ... end instead of if build.stable? then ... end.

so I thought that the block style might be preferred even for non-`patch` calls e.g. inside of `install`. What do you think?

Reply:
yeah, from what I see, `stable do` for the resource block, while `if build.stable?` is used for the build block.